From fdfde3b48ffe927214e9404f9b37e6aa91318929 Mon Sep 17 00:00:00 2001
From: Rajesh Bhagat <rajesh.bhagat@nxp.com>
Date: Thu, 15 Feb 2018 00:01:09 +0530
Subject: [PATCH 13/31] armv8: ls1043adcm: disable PPA loading during SPL stage
 for SD boot

PPA loading during SPL stage is not required for nornal
SD boot scenario.

Signed-off-by: Yangbo Lu <yangbo.lu@nxp.com>
Signed-off-by: Rajesh Bhagat <rajesh.bhagat@nxp.com>
---
 configs/ls1043adcm_sdcard_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configs/ls1043adcm_sdcard_defconfig b/configs/ls1043adcm_sdcard_defconfig
index e603184f3c..dc228e8584 100644
--- a/configs/ls1043adcm_sdcard_defconfig
+++ b/configs/ls1043adcm_sdcard_defconfig
@@ -3,7 +3,6 @@ CONFIG_TARGET_LS1043ADCM=y
 CONFIG_SPL_LIBCOMMON_SUPPORT=y
 CONFIG_SPL_LIBGENERIC_SUPPORT=y
 CONFIG_FSL_LS_PPA=y
-CONFIG_SPL_FSL_LS_PPA=y
 CONFIG_SPL_MMC_SUPPORT=y
 CONFIG_SPL_SERIAL_SUPPORT=y
 CONFIG_SPL_DRIVERS_MISC_SUPPORT=y
-- 
2.17.1

